_plugins/fractal_component.rb

```diff
@@ -0,0 +1,52 @@
+require 'open-uri'
+
+module Jekyll
+  class FractalComponentTag < Liquid::Tag
+    BASE_URL_ENV_VAR = 'FRACTAL_BASE_URL'
+
+    def initialize(tag_name, name, tokens)
+      super
+      @name = name.strip
+      @base_url = ENV[BASE_URL_ENV_VAR]
+      @fs_path = "node_modules/uswds/build/components/render/#{@name}.html"
```

Comment:
Is there any danger of this path changing, or components being nested in their own folders here based on how they're nested in the src folders?